tests/settling_game/00_realms_contract_test.py

```diff
@@ -159,19 +159,19 @@ async def test_mint(game_factory, number_of_tokens, tokens):
     )
 
     # check transfer
-    execution_info = await realms.balanceOf(accounts[0].contract_address).invoke()
+    execution_info = await realms.balanceOf(accounts[0].contract_address).call()
     print(f'Realms Balance for owner is: {execution_info.result.balance}')
 
     # claim resources
     await signer.send_transaction(
         account=accounts[0], to=resources_logic.contract_address, selector_name='claim_resources', calldata=[*uint(5042)]
     )
 
-    player_resource_value = await resources.balanceOf(accounts[0].contract_address, 1).invoke()
-    player_resource_value_1 = await resources.balanceOf(accounts[0].contract_address, 2).invoke()
-    player_resource_value_2 = await resources.balanceOf(accounts[0].contract_address, 3).invoke()
-    player_resource_value_3 = await resources.balanceOf(accounts[0].contract_address, 4).invoke()
-    player_resource_value_4 = await resources.balanceOf(accounts[0].contract_address, 5).invoke()
+    player_resource_value = await resources.balanceOf(accounts[0].contract_address, 1).call()
```

Comment:
Why the change from `invoke` to `call`? Since `balanceOf` doesn't modify state, I think there's not harm in keeping it `invoke`, especially because `invoke` is faster than `call` in the test framework.

Reply:
Ok, this was just me being StarkNoob, I thought it was the other way around... Wil change